docs/integrations/hosts-operating-systems/opentelemetry/opentelemetry-collector-insights.md

```diff
@@ -0,0 +1,447 @@
+description: Learn about the Sumo Logic OpenTelemetry Collector Insights app.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic OpenTelemetry Collector Insights app provides comprehensive monitoring and observability for your OpenTelemetry Collector instances. Monitor collector performance, telemetry data flow, resource utilization, and troubleshoot data collection issues with preconfigured dashboards and alerts. Track metrics and logs to ensure your telemetry pipeline is running smoothly and efficiently.
+
+This app supports OpenTelemetry Collector version `0.130.1-sumo-0` and later versions.
+
+We use the OpenTelemetry collector's built-in internal telemetry capabilities to collect metrics and logs about the collector itself. By default, the Collector exposes its own telemetry through internal metrics (via Prometheus interface on port 8888) and logs (emitted to stderr). The collector can also be configured to export its own telemetry data (metrics and logs) to Sumo Logic through OTLP/HTTP endpoints.
+
+:::info
+This app includes [built-in monitors](#opentelemetry-collector-insights-alerts). For details on creating custom monitors, refer to [Create monitors for OpenTelemetry Collector Insights app](#create-monitors-for-opentelemetry-collector-insights-app).
+:::
+
+## Fields creation in Sumo Logic for OpenTelemetry Collector Insights
+
+Following are the [fields](/docs/manage/fields/) which will be created as part of OpenTelemetry Collector Insights app installation, if not already present.
+
+- `sumo.datasource`. Has fixed value of `otel_collector`.
+- `_contentType`. Has fixed value of `OpenTelemetry`.
+- `deployment.environment`. User configured. Enter a name to identify your deployment environment.
+
+## Prerequisites
+
+### For OTLP endpoint configuration
+
+Before configuring the OTEL Collector integration, ensure you have the following prerequisites in place:
+
+1. **Sumo Logic OTLP Source**: You need to create an OTLP source in your Sumo Logic hosted collector. The OTLP source will provide the endpoint URL that the OTEL Collector will use to send telemetry data.
+
+  **Documentation**: [Creating a Sumo Logic OTLP Source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/otlp/)
+
+### For metrics collection
+
+The OpenTelemetry Collector must be configured to export its own metrics using the built-in telemetry capabilities. This requires:
+- OpenTelemetry Collector version 0.130.1-sumo-0 or later
+- Collector configured with telemetry metrics enabled at `detailed` level
+- Access to OTLP endpoint for metrics export
+- Internal metrics exposed on port 8888 (default)
+
+### For logs collection
+
+The OpenTelemetry Collector must be configured to export its own logs using the built-in telemetry capabilities. This requires:
```

Comment:
Can we change the language to make clear that debug log level and and json encoding are automatically set in the template which user will download